### PR TITLE
image for atom/node that fills up coursier cache

### PR DESCRIPTION
**Pull request by hedefalk**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +1,59 @@
 # https://docs.docker.com/reference/builder/
 # NOTE dockerignore is ignored https://github.com/docker/docker/issues/9455
 
-FROM ensime/ensime:v1.x
-MAINTAINER Sam Halliday, sam.halliday@gmail.com
+FROM debian:jessie
```
Comment (fommil) [inline Dockerfile:4]: do you not want to base off one of the existing ensime images? Basing off `ensime/ensime:v2.x` would seem sensible given that you want Java 7.

BTW, we'll have the coursier cache in `ensime/ensime:latest` soon when https://github.com/ensime/ensime-server/pull/1312 is unblocked.

Comment (hedefalk) [inline Dockerfile:4]: Well, in that case, I could just use that one instead :)  Coursier cache and java was all I needed just now, but I guess later I'll try to have at least smoketests for the whole stack like Emacs - you do sbt gen-ensime and everything on small projects, no?

Comment (fommil) [inline Dockerfile:4]: actually, I'd be happy for you to add a line to the `ensime/ensime:latest` run that downloads coursier and runs the server update script to populate the caches. Do you want to just do that?

Comment (hedefalk) [inline Dockerfile:4]: Sure!

**Pull request closed without merging**